### PR TITLE
Add cargo eden attributes

### PR DESCRIPTION
**Pull request by SilentSpike**
Inspired by #4283

- Add an `ace_cargo_space` attribute to vehicles to alter how much cargo they can carry
- Add an `ace_cargo_size` attribute to objects to alter how much cargo space they consume
- Add a `ace_cargo_setSize` public function
- Add a `ace_cargo_setSpace` public function
- Deprecate `ace_cargo_makeLoadable` public function and associated module

Known issues in current WIP implementation:
- [x] Public functions do not provide menus for future JIP players
  - [x] This is no different from current behaviour of existing public function, probably because it's a PITA to handle. Could be fixed using a `CBA_fnc_globalEventJIP`, would need to track the ID of which on the object for future function calls to overwrite the event.
- [x] Public functions used more than once on the same object (which doesn't usually have a menu) will add two action menu entries
- [x] Vehicles may have 2 cargo actions in their interaction menu (CBATeam/CBA_A3#567)

### CODE REVIEW

File: addons/cargo/CfgEden.hpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                        typeName = "NUMBER";
+                    };
+                    class GVAR(size) {
+                        displayName = CSTRING(size_edenName);
+                        tooltip = CSTRING(size_edenDesc);
+                        property = QGVAR(size);
+                        control = "Edit";
+
+                        // Expression only runs on the server, must handle actions for all machines and future JIPs (Why BI?!)
+                        expression = QUOTE(_this setVariable [ARR_3('%s',_value,true)]; [QQGVAR(EdenObjectInit),_this] call CBA_fnc_globalEventJIP;);
```
Comment (SilentSpike) [inline addons/cargo/CfgEden.hpp]: Just realised there's a missing `ARR_2` here that I must have added on my local version after committing. Rest assured, that's not what was making the `initObject` call fail to work.

**Comment by SilentSpike** [comment]
Good call, it actually looks like this is being handled already in `fnc_canLoadItemIn` so it can be removed from the action conditions :smile:

**Comment by SilentSpike** [comment]
Fixed and tested JIP compatiblity :+1: Working as expected

**Comment by SilentSpike** [comment]
Think this is now ready to go if anyone wants to review it. Assigned commy as he's a listed maintainer and pabst because he showed some interest in this.

**Comment by commy2** [comment]
I like it, could be merged.

**Comment by SilentSpike** [comment]
Mentioned in private slack, but looking at this now I'm not sure I like the way I've handled the locality in the public functions. I believe my intent was to allow the size to be set independently on each machine if desired (for example maybe only a specific player should be able to load the object in question). However it introduces a layer of complexity I'm not sure we want to have.

**Comment by PabstMirror** [comment]
I think size vars should to be the same on all machines

**Comment by SilentSpike** [comment]
Yeah I think it keeps things simpler to have values synced. I'll modify the public functions in this.

**Pull request merged**